app/src/main/java/ch/sdp/vibester/activity/GameSetupActivity.kt

```diff
@@ -60,7 +85,7 @@ class GameSetupActivity : AppCompatActivity(), AdapterView.OnItemSelectedListene
         return 1
     }
 
-    override fun onNothingSelected(parent: AdapterView<*>) {text = "One"}
+    override fun onNothingSelected(parent: AdapterView<*>) {text = "One"; difficulty = "Easy"}
```

Comment:
Can this function be tested if it won't take you much time?

Reply:
It kinda requires the app to crash as the user opens a spinner up. Basically, this function only gets called when there is no selection possible when the user tries to select something, however spinners are created with the first item selected by default. Testing this would require way too much effort for what it is worth, unfortunately